### PR TITLE
feat: add repeat / chime feature to timer app

### PR DESCRIPTION
**Pull request by thiswillbeyourgithub**
TODO :
* ~~if this gets approved, this page should be updated : https://wasp-os.readthedocs.io/en/latest/apps.html#timer-application~~
* ~~without forgetting to update the screenshots~~

### CODE REVIEW

File: wasp/apps/timer.py [hunk 1]
```diff
@@ -14,7 +14,6 @@
 
 import wasp
 import fonts
-import time
```
Comment (thiswillbeyourgithub) [inline wasp/apps/timer.py:17]: this import was not needed so I removed it

**Comment by thiswillbeyourgithub** [comment]
The reason the test is failing is `overflowed by 124 bytes`. This could be maybe solved by the PR that merges disaBLE to the settings?

**Review by daniel-thompson — CHANGES_REQUESTED** [review]
A patch description explaining what this change does and why would be really useful.

In general what is the purpose of the "Repeat" checkbox? In other words why does the user need to make a decision. Can't we just pick the "best" behaviour all the time?

**Comment by daniel-thompson** [comment]
> The reason the test is failing is `overflowed by 124 bytes`. This could be maybe solved by the PR that merges disaBLE to the settings?

Let's see what happens when this comes back for re-testing. It's possible replacing Game of Life with the Gallery app will have created some extra space.

**Comment by fgaz** [comment]
While the chime feature is great, I often like my watch to be as silent as possible. Maybe the chime could be optional?

Or even better, there could be a global setting that has 3 volume options (completely silent, vibration, sound) that applies to _all_ apps (well, the timer and the alarm)

**Comment by thiswillbeyourgithub** [comment]
> While the chime feature is great, I often like my watch to be as silent as possible. Maybe the chime could be optional?
> 
> Or even better, there could be a global setting that has 3 volume options (completely silent, vibration, sound) that applies to _all_ apps (well, the timer and the alarm)

I don't understand. To me, the goal of the timer app is to be notified that X time has passed. I don't see the point of a silent timer app.

**Comment by thiswillbeyourgithub** [comment]
I'm thinking of working on a pomodoro timer app, largely based on this fork. Maybe you would prefer having two separate apps and leaving this one as is?

**Comment by thiswillbeyourgithub** [comment]
In the end I don't think this PR is a good idea, seeing the progress i made in #320

**Pull request closed without merging**